### PR TITLE
Suggested export options

### LINKED ISSUE #17: recommended export options
Add a screen shot with recommended export options

### PR DESCRIPTION
**Pull request by PatrickRWright**
Pretty straight forward function to help people troubleshoot why "things" may not be working or train people what type of exports to request.

I would like to add the function call after `read_secuTrial_raw()` in `read_secuTrial()` to warn people if they use the wrapper. What do you think @aghaynes @markomi ?

Also I realized that we do not seem to be tracking the export format currently since we assume its always either "CSV format" or "CSV format for MS Excel". I think we should track this export option and catch it if e.g. SAS, CDISC, SPSS is specified. I have added this to the PR too.

closes #17

### CODE REVIEW

**Comment by PatrickRWright** [comment]
Adding `check_export_options()` to `read_secuTrial()` created quite a few warnings in the test suite (as expected). This should not be too hard to remedy. However, before I do that I would like to know if you both agree with running the check.

**Comment by aghaynes** [comment]
> we do not seem to be tracking the export format currently since we assume its always either "CSV format" or "CSV format for MS Excel". I think we should track this export option and catch it if e.g. SAS, CDISC, SPSS is specified.

The first part (tracking the format) we certainly do... this gets the file extension which is saved in export_options as `extension` https://github.com/SwissClinicalTrialOrganisation/secuTrialR/blob/385e5dc502d236c0e0c1005d59edb62af0d3f522/R/read_export_options.R#L143 I dont remember whether we check it though... 

Seems like a reasonable idea on the whole though

**Comment by PatrickRWright** [comment]
@aghaynes so any changes, or can I go ahead and adjust the tests accordingly so we don't have 24 warnings?

File: README.md [hunk 1]
```diff
@@ -12,6 +13,20 @@ An R package to handle data from the clinical data management system (CDMS) [sec
 devtools::install_github("SwissClinicalTrialOrganisation/secuTrialR")
 ```
 
+## Recommended export options
+
+While the package strives to allow loading of as many types of secuTrial data exports
+as possible, there are certain export options which are less likely to cause issues.
+If possible it is suggested to export data which adheres to a suggested option set.
+Thus, we suggest to work with exports which:
+- are zipped
```
Comment (markomi) [inline README.md]: If I'm not mistaken, we do not write explicitly inside of the readme that the CSV export format is required. Shall we add this here?

**Comment by markomi** [comment]
For me the addition of the warnings to read_secuTrial() would be fine. I'd also be OK with converting the warning to a message, if you prefer.

**Comment by aghaynes** [comment]
Its ok with me

**Comment by markomi** [comment]
@PatrickRWright should I merge now, or do you want to add something?

File: README.md (hunk 1 above)
Comment (PatrickRWright) [reply README.md]: yes, good idea.

**Comment by PatrickRWright** [comment]
I will go over the tests and remove the warnings if possible.

Also, how about having "English" as suggested language?

**Comment by PatrickRWright** [comment]
I'm done.

**Comment by markomi** [comment]
Thanks @PatrickRWright! This will save us a lot of troubleshooting time. I'm merging.

**Pull request merged**